src/meshlab/glarea.cpp

```diff
@@ -2014,6 +2014,52 @@ bool GLArea::viewFromFile()
     return true;
 }
 
+bool GLArea::saveViewToFile()
+{
+    QFileDialog* saveDiag = new QFileDialog(this,tr("Save View To File"), "./", tr("View file (*.xml)"));
```

Comment:
Memory leak? You never free this...

Reply:
Good point. I fixed it. Here's where I copied the logic from, which may need fixing too: https://github.com/cnr-isti-vclab/meshlab/blob/cd35efd06fedc77ff6fb0f4b28e32007f1af3d62/src/meshlab/mainwindow_RunTime.cpp#L2148

It is not strictly speaking a memory leak, as it is a child of "this", the parent, so when the parent quits the child will be deallocated. https://doc.qt.io/qt-5/objecttrees.html

That being said, likely each time one opens that dialog the memory being held increases a little, so at some level it is a leak, even if it is tracked by the parent.